securesystemslib/keys.py

```diff
@@ -544,7 +544,8 @@ def format_metadata_to_key(key_metadata):
 
   # Convert 'key_value' to 'securesystemslib.formats.KEY_SCHEMA' and generate
   # its hash The hash is in hexdigest form.
-  default_keyid = _get_keyid(keytype, scheme, key_value)
+  if (default_keyid == None):
```

Comment:
According to [PEP 8](https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations) the correct way to check if a condition is `None` is with the identity operator `is`:
```suggestion
  if default_keyid is None:
```